metagraph/src/cli/align.cpp

```diff
@@ -319,6 +308,58 @@ std::string format_alignment(std::string_view header,
 
             sout += fmt::format("{}\n", Json::writeString(builder, json_line));
         }
+    } else if (config.output_gaf) {
+        std::string base = fmt::format("{}\t{}\t", header, paths.get_query().size());
+        for (size_t i = 0; i < paths.size(); ++i) {
+            const auto &path = paths[i];
```

Comment:
```suggestion
        for (const auto &path : paths) {
```

Reply:
I had it set up this way because a bug in GCC-8 was causing the compilation to crash because of this kind of for-loop. Let's see if it happens this time.